### PR TITLE
fix(localisation): Fix the nggettext_extract for ts files (SDESK-3812)

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: scripts/apps/tansa/index.ts [hunk 1]
```diff
@@ -4,12 +4,15 @@
...
+// tansa server base urls e.g 'http://localhost:8080/tansaclient/'
+var baseUrl = `${location.protocol}//${location.host}/tansaclient/`;
+var tansaScript =
+`
     <script type="text/javascript">
         var tansa = {
             settings: {
-                // tansa server base urls e.g 'http://localhost:8080/tansaclient/'
-                baseUrl: '${location.protocol}//${location.host}/tansaclient/',
+                baseUrl: '<baseUrl>',
```
Comment (tomaskikutis) [inline scripts/apps/tansa/index.ts]: How is this related to `nggettext_extract`? I see no `gettext` tags here.

Comment (ioanpocol) [reply scripts/apps/tansa/index.ts]: nggettext_extract was failing to parse the file:

ivp:superdesk-client-core ivp$ grunt nggettext_extract
Running "nggettext_extract:pot" (nggettext_extract) task
Error parsing scripts/apps/tansa/index.ts at line 5 column 26

Comment (tomaskikutis) [reply scripts/apps/tansa/index.ts]: Did it fail in the command line when doing the extraction? Will it be fine now for all other files?

File: scripts/apps/tansa/index.ts [hunk 2]
```diff
@@ -28,7 +31,11 @@ document.write(`
             $rootScope.$broadcast('tansa:end', isCancelled);
         }
     </script>
-`);
+`;
+
+// workaround to avoid nggettext_extract parsing error
+tansaScript = tansaScript.replace('<baseUrl', baseUrl);
```
Comment (tomaskikutis) [inline scripts/apps/tansa/index.ts]: I think you are missing a `>` in the string.

Comment (ioanpocol) [reply scripts/apps/tansa/index.ts]: Fixed

File: scripts/apps/tansa/index.ts (hunk 1 above)
Comment (ioanpocol) [reply scripts/apps/tansa/index.ts]: Only this file had problems on nggettext_extract parsing

>ivp:superdesk-client-core ivp$ grunt nggettext_extract
Running "nggettext_extract:pot" (nggettext_extract) task
>
>Done.

File: scripts/apps/tansa/index.ts (hunk 2 above)
Comment (tomaskikutis) [reply scripts/apps/tansa/index.ts]: screenshot_7

Did I mention that these "Fixed" and "Done" comments are annoying? :smiley: Use the resolve button.
